PterodactylLib/Classes/Pterodactyl.swift

```diff
@@ -33,29 +33,39 @@ public class Pterodactyl {
     }
     
     public func triggerSimulatorNotification(withFullPayload payload: [String: Any]) {
-        let endpoint = "http://\(host):\(port)/simulatorPush"
-        
+        let endpoint = "http://\(host):\(port)/\(pushEndpoint)"
+
         guard let endpointUrl = URL(string: endpoint) else {
             return
         }
-        
+
         //Make JSON to send to send to server
-        var json = [String:Any]()
-        json["simulatorId"] = ProcessInfo.processInfo.environment["SIMULATOR_UDID"]
-        json["appBundleId"] = targetAppBundleId
-        json["pushPayload"] = payload
-        
-        guard let data = try? JSONSerialization.data(withJSONObject: json, options: []) else {
-            return
-        }
+        let json = PushRequest(
+            simulatorId: ProcessInfo.processInfo.environment["SIMULATOR_UDID"] ?? "booted",
+            appBundleId: targetAppBundleId,
+            pushPayload: JSONObject(payload)
+        )
```

Comment:
Ultimately, being able to share request definitions between the library and server seems like a big win.